Dockerfile

```diff
@@ -0,0 +1,48 @@
+ARG GO_VERSION=1.18
+ARG $=v1.1.3
```

Comment:
Where is this used?

Reply:
this has to be a RUNC_VERSION environment variable, my bad.